src/components/Paymethods.tsx

```diff
@@ -0,0 +1,57 @@
+import { number } from "io-ts";
+import { Integer } from "italia-ts-commons/lib/numbers";
+import React, { useState } from "react";
+import { TabContent, TabPane, Nav, NavItem, NavLink } from "reactstrap";
+import PaymentMethodTabPane from "./PaymentMethodTabPane";
+import { format, parseISO } from "date-fns";
+import { useTranslation } from "react-i18next";
+import "./Paymethods.css";
+import { PaymentMethod } from "../generated/definitions/PaymentMethod";
+import classNames from "classnames";
+
+function Paymethods(props) {
```

Comment:
The `prop` type is mandatory.

```suggestion
type PaymethodsProps = {
  paylist: ReadonlyArray<PaymentMethod>;
};

function Paymethods(props: PaymethodsProps) {
```